src/browser_app_skeleton/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Allow: *
```

Comment:
I've done a bit of research and it looks like `Disallow` is better supported

```suggestion
Disallow:
```

Thoughts on changing to that?

Reply:
That seems confusing.  Maybe we can add a comment.

Reply:
Yeah it is super weird. Comment sounds good 👍

Reply:
Yeah really it shouldn't need either afaik, but changing to disallow could work fine